web_widget_open_tab/static/src/js/widget.js

```diff
@@ -0,0 +1,46 @@
+odoo.define('web_widget_open_tab.FieldOpen', function(require) {
+    "use strict";
+
+    var AbstractField = require('web.AbstractField');
+    var field_registry = require('web.field_registry');
+    var ListRenderer = require('web.ListRenderer');
+
+    var FieldOpen = AbstractField.extend({
+        description: "",
+        supportedFieldTypes: ['integer'],
+        events: _.extend({}, AbstractField.prototype.events, {
+            'click': '_onClick',
+        }),
+        isSet: function () {
+            return true;
+        },
+        _getReference: function () {
+            return 'model=' + this.model + '&id=' + this.res_id;
+        },
+        _renderReadonly: function () {
+            var $content = $(
+                '<a href="#' + this._getReference() + '">'
```

Comment:
Using `<a href="#' + this._getReference() + '" onclick="window.open(this.href);">` will effectively open in a new tab. This looks more intuitive to me, isn't it?

Reply:
I implemented the same idea but on the `onclick` function. Just to simplify it :smile: